opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py

```diff
@@ -401,6 +393,23 @@ def __init__(
         export_interval_millis: Optional[float] = None,
         export_timeout_millis: Optional[float] = None,
     ) -> None:
+        # Prioritize exporter level configuration
```

Comment:
Can we just remove `preferred_temporality` and `preferred_aggregation` constructor params altogether from this class, and instead always read it off the exporter?

Reply:
> It would be confusing to me as a user to be able to configure these for other MetricReaders

What is the confusion there? `PeriodicExportingMetricReader` is always paired with a push exporter which is the actual thing that can support different temporalities. For `preferred_aggregation`, some exporters may not support certain aggregations (e.g. ExponentialHistogram) so IMO it makes sense to configure it at the exporter level rather than the `PeriodicExportingMetricReader`. 

> Also, I don't think having the params on `PeriodicExportingMetricReader` is a big deal. Any strong reasons against it?

I think we are providing too many ways to configure it. Idk if we need all of this flexibility. There are environment variables and then two levels of constructors

Reply:
Sure. I deferred the configuration to exporter for `PeriodicExportingMetricReader`